### DISCUSSION TITLE
Did you get inspiration from octo.nvim?

### DISCUSSION THREAD

**Discussion by molleweide** [ldelossa/gh.nvim#33] [General]
Hi! I am looking forward to learning and helping out with this plugin.

Quick question: Did you use `octo.nvim` before? If so, why did you decide to write a new plugin apart from the arguments given in the readme?

**Comment by ldelossa** [comment]
Hey, thanks. 

Before https://github.com/pwntester/octo.nvim/issues/98#issuecomment-1128158030

and 

https://github.com/pwntester/octo.nvim/issues/288#issuecomment-1128161986

These were major limitations for me, but have this been implemented. 

The other major difference is "gh.nvim" pulls the remote code local and checks out the branch/commits. I wanted this because it creates a seamless review process. All LSP tools work and you can run tests and local dev environments to confirm the code in the pull works well. 

I also have started to quite like the thread and issue buffers. 


Moving forward, I plan to keep removing my usages of GitHub in the browser and place them into gh.nvim. The next big aspect is the "Notifications" features.

**Reply by molleweide** [reply]
Allright! Thanks for giving me those threads and explaining.
